Author: Agustin Henze <tin@sluc.org.ar>
Description: Remove unneeded dependencies.

--- configparser-3.2.0r3.orig/setup.py
+++ configparser-3.2.0r3/setup.py
@@ -31,8 +31,6 @@ setup (
     py_modules = ('configparser', 'configparser_helpers'),
     zip_safe = True,
     install_requires = [
-        'ordereddict',
-        'unittest2',
     ],
     classifiers = [
         'Development Status :: 4 - Beta',
